packages/@sanity/form-builder/src/inputs/arrays/ArrayOfPrimitivesInput/ArrayOfPrimitivesInput.tsx

```diff
@@ -25,16 +24,48 @@ function move<T>(arr: T[], from: number, to: number): T[] {
   return copy
 }
 
-function insertAt<T>(arr: T[], index: number, item: T): T[] {
+/**
+ * @param index index to insert item after. An index of -1 will prepend the item
+ * @param arr the array to insert the item into
+ * @param item the item to insert
+ *
+ * example:
+ *  insertAfter(-1, ["one", "two"], "hello")
+ *  => ["hello", "one", "two"]
```

Comment:
🤓/🍬  you could wrap this in markdown and add a `js` language hint for syntax highlighting when you hover